cmd/chihaya/config.go

```diff
@@ -21,15 +21,27 @@ type hookConfig struct {
 	Config interface{} `yaml:"config"`
 }
 
+type hookConfigs []hookConfig
+
+// Names returns all hook names listed in the configuration.
+func (hookCfgs hookConfigs) Names() (hookNames []string) {
+	hookNames = make([]string, 0)
```

Comment:
Adding that change makes the output look like this:


![screen shot 2017-06-02 at 8 49 12 pm](https://cloud.githubusercontent.com/assets/4852029/26749647/1c3d1c7a-47d5-11e7-912f-f6dca30dd485.png)